src/LogFileReader.cpp

```diff
@@ -32,6 +32,13 @@
 
 #include "LogFileReader.h"
 
+namespace
+{
+    const QString TIME_FORMAT = "hh:mm:ss.zzz";
+    QString TEMP_TIME;
```

Comment:
The namespace should be reserved for constants, so this variable would better off being initialized in the function it's being used in.